### DISCUSSION TITLE
Add names to FOEN support

### DISCUSSION THREAD

**Discussion by hammerdirt-analyst** [hammerdirt-analyst/IQAASL-End-0f-Sampling-2021#39] [General]
Is there a reason why we can't put Sam and Amandas name on the title page?

**Comment by AmandaFinger** [comment]
It would make a lot of sense to add Sam and me to the "Impressum" in addition to Rita Barros. There is no need to add us as co-authors, though. Thanks a lot!

**Reply by hammerdirt-analyst** [reply]
Ok this was done in commit #40
